SpatialGDK/Source/SpatialGDKServices/Private/Interop/Connection/EditorWorkerController.cpp

```diff
@@ -37,20 +37,26 @@ struct EditorWorkerController
 
 		PIEEndHandle = FEditorDelegates::PrePIEEnded.AddRaw(this, &EditorWorkerController::OnPrePIEEnded);
 
-		int32 PlayNumberOfServers;
-		GetDefault<ULevelEditorPlaySettings>()->GetPlayNumberOfServers(PlayNumberOfServers);
+		const ULevelEditorPlaySettings* LevelEditorPlaySettings = GetDefault<ULevelEditorPlaySettings>();
+		const int32 WorkerCount = LevelEditorPlaySettings->GetTotalPIEServerWorkerCount();
+		WorkerIds.SetNum(WorkerCount);
+		ReplaceProcesses.SetNum(WorkerCount);
```

Comment:
True it might not, but I would argue this is the best thing to do since it's the minimum upper bound preventing reallocation. 

Also just found a redundant `ReplaceProcesses.empty()` at the start of the method which I've now deleted.